packages/callbacks/tizen/inc/tizen_interop_callbacks_plugin.h

```diff
@@ -0,0 +1,66 @@
+#ifndef FLUTTER_PLUGIN_TIZEN_INTEROP_CALLBACKS_PLUGIN_H_
+#define FLUTTER_PLUGIN_TIZEN_INTEROP_CALLBACKS_PLUGIN_H_
+
+#include <dart_api.h>
+#include <flutter_plugin_registrar.h>
+
+#include <functional>
+#include <map>
+
+#ifdef FLUTTER_PLUGIN_IMPL
+#define FLUTTER_PLUGIN_EXPORT __attribute__((visibility("default")))
+#else
+#define FLUTTER_PLUGIN_EXPORT
+#endif
+
+#if defined(__cplusplus)
+extern "C" {
+#endif
+
+FLUTTER_PLUGIN_EXPORT void TizenInteropCallbacksPluginRegisterWithRegistrar(
+    FlutterDesktopPluginRegistrarRef registrar);
+
+#if defined(__cplusplus)
+}  // extern "C"
+#endif
+
+struct CallbackInfo {
+  void *callbackPtr;
+  void *actualUserData;
+};
+
+typedef std::function<void()> CallbackWrapper;
+void RequestCallbackCall(CallbackWrapper *wrapper);
+
+// A few structures copied from Tizen Native API headers, as we need whole
+// definion if it is used in callback signature.
+typedef struct {
+  double latitude;
+  double longitude;
+} location_coords_s_copy;
```

Comment:
Only these 4 structs (and not more) used as callback arguments?

Reply:
those 4 are used directly, there are many others that are passed with pointer so we don't need full definition